frontend/src/components/IntegrationConfiguration.tsx

```diff
@@ -468,9 +485,6 @@ export const IntegrationConfig: FC<{ roadmapId: number; name: string }> = ({
               <SelectBoard configuration={configuration} />
             </>
```

Comment:
```suggestion
              <SelectBoard configuration={configuration} />
              <Remove configuration={configuration} />
            </>
```

The remove button could be moved in to the same group, as it has the same precondition.